release.sh

```diff
@@ -258,5 +255,6 @@ echo " - Edit the following file https://github.com/eclipse/rdf4j-doc/blob/maste
 echo " - Go to https://github.com/eclipse/rdf4j-doc/tree/master/site/content/news and create rdf4j-${NEWS_FILE_NAME}.md"
 echo " - Post to Google Groups: https://groups.google.com/forum/#!forum/rdf4j-users"
 echo "     - Good example: https://groups.google.com/forum/#!topic/rdf4j-users/isrC7qdhplY"
-echo " - Upload the javadocs " #TODO Where should they be uploaded and how?
+echo " - Upload the javadocs by adding them to rdf4j-doc project: site/static/javadoc/${MVN_VERSION_RELEASE}"
```

Comment:
The location of the generated javadocs, is it: assembly/target/site/apidocs

Reply:
Actually it's target/site/apidocs (in the project root) now. Or, of course, you can get them by unzipping the SDK zip :)

Reply:
Could you add that as an echo line to the script? In the future we could automate that too.